test/sysconfig_test.rb

```diff
@@ -9,6 +9,23 @@
   subject(:sysconfig) do
     new_sysconfig(configfiles)
   end
+  let(:configfiles) { ["#{DATA_PATH}/sysconfig/*"] }
+
+  before do
+    allow(Yast::SCR).to receive(:Execute).with(path(".target.bash_output"), anything)
+      .and_return("exit" => 0, "stdout" => "", "stderr" => "")
+    # run git version of parse_configs.pl
+    allow(Yast::SCR).to receive(:Execute).with(path(".target.bash"), /parse_configs.pl/) do |path, command|
+      params = command.split
+      params[0] = SRC_PATH + "/bin/parse_configs.pl"
```

Comment:
NP: `File.join` would be a  bit better...
```suggestion
      params[0] = File.join(SRC_PATH, "bin/parse_configs.pl")
```